templates/lp12-widget/template/README.md

```diff
@@ -0,0 +1,91 @@
+# <%=widget_title%>
+
+<%=widget_description%>
+
+## Information
+
+| name                  | version           | bundle           |
+| ----------------------|:-----------------:| ----------------:|
+| <%=widget_name%>      |<%=widget_version%>| Launchpad        |
+
+## Widget Checklist
+
+ - [ ] Fault Tolerance: Widget gracefully behaves/fails with loss of connection.
+ - [ ] Fault Tolerance: Widget gracefully fails if session is lost.
+ - [ ] Fault Tolerance: Widget gracefully and productively handles error responses.
+ - [ ] Extensibility: Look and feel is manageable via theming.
+ - [ ] Security: Secure from XSS.
+ - [ ] Security: Secure from CSRF.
+ - [ ] Accessibility: Support for color blind users.
+ - [ ] Accessibility: Support for users with motor-inability (keyboard navigation).
+ - [ ] Accessibility: Support for users who are blind (screen reader).
+ - [ ] i18n: All UI messages are externalized and localizable.
+ - [ ] i18n: All dates and numbers are localized.
+ - [ ] i18n: Works RTL.
+ - [ ] Mobile: SDK compatible.
+ - [ ] Mobile: Widget is responsive to mobile & tablet.
+ - [ ] Documentation: Reference files linked from README.
+ - [ ] Documentation: Dependencies (bower & UI components used) listed in README.
+ - [ ] Documentation: Modules/classes JSDoc.
+ - [ ] Documentation: Widget feature list documented.
+ - [ ] Testing: Distribution folder
+
+## Dependencies
+
+* [base][base-url]
+* [core][core-url]
+* [ui][ui-url]
+
+## Dev Dependencies
+
+* [angular-mocks ~1.2.28][angular-mocks-url]
+* [config][config-url]
+
+## Preferences
+- List of widget preferences
+
+## Events
+- List of event the widget publishes/subscribes
+
+## Custom Components
+- list of widget custom components (if any)
+
+## Develop Standalone
+
+<!-- ```bash
+git clone <repo-url> && cd <widget-path>
+bb install -l
```

Comment:
There's no `bb install`, I see it's messed-up with `bblp`. Also, I would expect to have some info about installing CLI tools first.